File: swift/difference-of-squares/Sources/DifferenceOfSquares/DifferenceOfSquares.swift
struct Squares {
    let squareOfSum: Int
    let sumOfSquares: Int
    let numbers: [Int]
    var differenceOfSquares: Int {
        squareOfSum - sumOfSquares
    }
    
    init(_ numbersCount: Int) {
        self.numbers = (1...numbersCount).map({ $0 })
        let sum = self.numbers.reduce(0, +)
        self.squareOfSum = sum * sum
        self.sumOfSquares = self.numbers.map({ $0 * $0 }).reduce(0, +)
    }
}

File: python/difference-of-squares/_iterations/1/difference_of_squares.py
def square_of_sum(number):
    return sum(range(number + 1)) ** 2


def sum_of_squares(number):
    return sum(number**2 for number in range(number + 1))


def difference_of_squares(number):
    return square_of_sum(number) - sum_of_squares(number)
